modules/log6x-input-spec-filter-audit-infrastructure.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-input-spec-filter-audit-infrastructure_{context}"]
+= Filtering the audit and infrastructure log inputs by source
+
+You can define the list of `audit` and `infrastructure` sources to collect the logs by using the `input` selector.
+
+.Prerequisites 
+
```

Comment:
This cannot be used here.

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

OCP does not allow hard links to internal content.
OCP requires xrefs for internal content.
OCP does not allow xrefs in modules, only assemblies.